libs/atqr/domain/src/lib/challenge-entity/challenge-entity.ts

```diff
@@ -44,15 +59,11 @@ export class Challenge {
     return this._deadline;
   }
   get price() {
-    if (this._price > 24) {
-      return this._price;
-    } else {
-      new Error('Selecione um valor acima de 25 reais');
-    }
     return this._price;
   }
-  get notifiedSupervisor() {
-    return this._notifiedSupervisor;
+
+  get statusSupervisor() {
```

Comment:
I believe this should be supervisorStatus, right? statusSupervisor sounds kinda weird. But I get that this is how we say it in Portuguese.